moped-database/migrations/1719582711923_refine_funding_source/up.sql

```diff
@@ -0,0 +1,507 @@
+DROP VIEW IF EXISTS component_arcgis_online_view;
+DROP VIEW IF EXISTS project_list_view;
+
+CREATE OR REPLACE VIEW project_list_view AS WITH project_person_list_lookup AS (
+    SELECT
+        mpp.project_id,
+        string_agg(DISTINCT concat(mu.first_name, ' ', mu.last_name, ':', mpr.project_role_name), ','::text) AS project_team_members
+    FROM moped_proj_personnel AS mpp
+    INNER JOIN moped_users AS mu ON mpp.user_id = mu.user_id
+    INNER JOIN moped_proj_personnel_roles AS mppr ON mpp.project_personnel_id = mppr.project_personnel_id
+    INNER JOIN moped_project_roles AS mpr ON mppr.project_role_id = mpr.project_role_id
+    WHERE mpp.is_deleted = false AND mppr.is_deleted = false
+    GROUP BY mpp.project_id
+),
+
+funding_sources_lookup AS (
+    SELECT
+        mpf_1.project_id,
+        string_agg(DISTINCT mfs.funding_source_name, ', ' ORDER BY mfs.funding_source_name) AS funding_source_name
```

Comment:
@mddilley -- Sorry if it's strange to tag you in a PR that I'm actively working on, but I wanted to ask you --

I'm not 100% sure you asked for this, but we did circle around the idea, at least, during our conversation yesterday. Do you like this change? For the existing funding column where it creates a comma-delimited list of funding sources names, do you like this change where i now am deduplicating them and ordering alphabetically?